syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -49,7 +49,16 @@ def get_class_attributes(self):
         for example precision_fractional is important when wrapping the result of a method
         on a self which is a fixed precision tensor with a non default precision_fractional.
         """
-        return {"precision_fractional": self.precision_fractional}
+        return {
+            "owner": self.owner,
+            "id": self.id,
+            "field": self.field,
+            "base": self.base,
+            "precision_fractional": self.precision_fractional,
+            "kappa": self.kappa,
+            "tags": self.tags,
+            "description": self.description,
+        }
```

Comment:
Oh, one thing I've not mentioned, you shouldn't provide owner and id in the get_class_attributes as you don't want tensors results to have the same id than the initial tensor and the owner should be inferred naturally.
Regarding tags and description, you may also want not to add them here.
Basically get_class_attributes is all the attribute of x you want to see in f(x), for any f

Reply:
Well for owner I'm not 100% sure maybe you can keep it

Reply:
So for .get we want to preserve these attributes. I will modify it to manually add those attributes